### PR TITLE
PIE-4766 json_format support

### PR DESCRIPTION
**Pull request by FinchPowers**
cc @MarcDufresne @ajutras 

Before opening a PR, make sure that, if relevant:
- [x] Changes are covered by automated tests.
- [x] New / updated functions / methods have typehints.
- [x] READMEs have been updated.
- [x] The affected parties are / will be notified of incoming breaking changes.

### CODE REVIEW

File: opset/configurator.py [hunk 1]
```diff
@@ -105,6 +105,7 @@ def _convert_type(value: str) -> Union[str, bool, int, float, Dict, List]:
         return value
 
     def _get_config_file_path(self, config_name: str) -> str:
+        print(self.config_path, config_name)
```
Comment (MarcDufresne) [inline opset/configurator.py]: print!

**Review by MarcDufresne — APPROVED** [review]

**Comment by FinchPowers** [comment]
@MarcDufresne @ajutras Btw I can't put you as reviewers since you are no longer part of the org. That is why I put you in cc. :)

File: opset/configurator.py (hunk 1 above)
Comment (FinchPowers) [reply opset/configurator.py]: Yes, removed! I added that while debugging weird unrelated issues.

**Review by JonnyLinton — APPROVED** [review]
🎉

**Comment by MarcDufresne** [comment]
Thanks for keeping us in the loop anyway! I was able to symbolically approve :P

**Review by alexj-novisto — APPROVED** [review]

**Pull request merged**